File: Assets/CarPositioning_System/storage.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class storage : MonoBehaviour {
	public Computer_Car_Script[] allcomputercars  ;
	public int part = 5;
	public int nexttouse = 0;
	public float halfheight = 1.000f;

	public GameObject playercar  ;
	public player_pos_gui playercarscript ;

	public GameObject enemycar  ;
	public Computer_Car_Script enemycarscript  ;

	//public var allcomputercars : Computer_Car_Script[];
	bool allcomputercars_ElementsExpand = true;
	int allcomputercars_ElementsSize  = 1;

	public GameObject lapsystemobject  ;
	public lap_script lapscript  ;
	public int menucamefrom = 0;

	
}

File: Assets/CarPositioning_System/storage.js
#pragma strict

public var part = 5;
public var nexttouse = 0;
public var halfheight = 1.000;

public var playercar : GameObject;
public var playercarscript : player_pos_gui;

public var enemycar : GameObject;
public var enemycarscript : Computer_Car_Script;

public var allcomputercars : Computer_Car_Script[];
var allcomputercars_ElementsExpand : boolean = true;
var allcomputercars_ElementsSize : int = 1;

public var lapsystemobject : GameObject;
public var lapscript : lap_script;
public var menucamefrom = 0;

function Start () {

}

function Update () {

}